README.md

```diff
@@ -215,6 +216,34 @@ This example shows how to enable the `data`, `orgs`, `metadata`, and `other` too
        }
 ```
 
+### Configure Tools
+
+The Salesforce DX MCP Server also supports registering individual **tools**. This can be used in combination with **toolsets** to further fine-tune registered tools.
+
+Use the `--tools` flag to specify tools when you configure the Salesforce DX MCP Server. Separate multiple tools with commas. The `--tools` flag is optional
+
+This example shows how to enable the `orgs` and `metadata` toolsets, as well as the `run_soql_query` and `run_apex_test` tools when configuring the MCP server for VS Code:
+
+```json
+       "servers": {
+         "Salesforce DX": {
+           "type": "stdio",
```

Comment:
Question (separate from the --tools flag, it's just a question I had while moving these docs to official site):  Is it necessary to include this flag?  I think it's the default behavior.  If so, for simplicity, I think we should remove it.  No need to do it in the README, I'll just do it in the "real" docs. Thx.